### PR TITLE
Add support for define.alias('foo', 'bar')

### PR DESCRIPTION
**Pull request by tmquinn**

### CODE REVIEW

File: lib/deps-from-ast.js [hunk 1]
```diff
@@ -40,19 +40,27 @@ module.exports = function depsFromAST(ast) {
...
+              if (value !== 'exports' && value !== 'require') {
+                imports.push(value);
+              }
             }
+            return STOP;
           }
-          return STOP;
         }
+      } else if (
+        entry.callee.object &&
```
Comment (rwjblue) [inline lib/deps-from-ast.js]: Can you replace this with `entry.callee.type === 'MemberExpression' &&`?  It makes checking for the presence of `entry.callee.object` unnecessary, and makes it clearer in the code what you are actually checking for...

Comment (tmquinn) [reply lib/deps-from-ast.js]: Nice, I did not like that anyway!

**Review by rwjblue — APPROVED** [review]
LGTM

@stefanpenner - Any objections?

**Comment by stefanpenner** [comment]
released as v1.4.0 🎉

**Pull request merged**